go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	k8s.io/mount-utils v0.32.3
 	k8s.io/pod-security-admission v0.32.0
 	k8s.io/utils v0.0.0-20250321185631-1f6e0b77f77e
-	sigs.k8s.io/cloud-provider-azure v1.29.1-0.20250418064547-4dfd672dab58
+	sigs.k8s.io/cloud-provider-azure v1.29.1-0.20250422032600-88c837448b9c
```

Comment:
Hi @andyzhangx , out of curious, if we bump the `sigs.k8s.io/cloud-provider-azure` by commit it will generate the most recent tag in the repo `cloud-provider-azure`. I tried it will use the v1.31.1 verion tag instead, how could we make it keep `v1.29.1`(is there some reason to keep it as `v1.29.1`)?
```console
$ go get sigs.k8s.io/cloud-provider-azure@88c837448b9cefe56b218236adce1445435199ff
go: upgraded sigs.k8s.io/cloud-provider-azure v1.29.1-0.20250418064547-4dfd672dab58 => v1.31.1-0.20250422032600-88c837448b9c
```

Reply:
`v1.29.1` is like a pseudo prefix, only the `20250418064547-4dfd672dab58` matters.

Reply:
Got it, thanks for the clarify!